skfda/ml/clustering/kmeans.py

```diff
@@ -702,7 +698,9 @@ def _check_params(self):
 
     def _compute_inertia(self, membership, centroids,
                          distances_to_centroids):
-        return np.sum(membership ** self.fuzzifier * distances_to_centroids ** 2)
+        return np.sum(
+            membership ** self.fuzzifier * distances_to_centroids ** 2,
```

Comment:
I usually do not leave spaces around the exponential operator, to easily see the terms in a sum or product. Do our recent linter complain about that?

Reply:
It is okay do to that.

Reply:
So do it, then.

Reply:
It should already be changed.

Reply:
Ok, I see it now